helix-term/src/keymap.rs

```diff
@@ -591,24 +591,24 @@ impl Default for Keymaps {
                 "t" => align_view_top,
                 "b" => align_view_bottom,
                 "m" => align_view_middle,
-                "k" => scroll_up,
-                "j" => scroll_down,
-                "b" => page_up,
-                "f" => page_down,
-                "u" => half_page_up,
-                "d" => half_page_down,
+                "k" | "up" => scroll_up,
+                "j" | "down" => scroll_down,
+                "b" | "S-up" => page_up,
+                "f" | "S-down" => page_down,
+                "u" | "C-up" => half_page_up,
+                "d" | "C-down" => half_page_down,
             },
             "Z" => { "View" sticky=true
                 "z" | "c" => align_view_center,
                 "t" => align_view_top,
                 "b" => align_view_bottom,
                 "m" => align_view_middle,
-                "k" => scroll_up,
-                "j" => scroll_down,
-                "b" => page_up,
-                "f" => page_down,
-                "u" => half_page_up,
-                "d" => half_page_down,
+                "k" | "up" => scroll_up,
+                "j" | "down" => scroll_down,
+                "b" | "S-up" => page_up,
+                "f" | "S-down" => page_down,
+                "u" | "C-up" => half_page_up,
+                "d" | "C-down" => half_page_down,
```

Comment:
Where was ctrl-up based on? I don't see neovim or kakoune have this behavior.

Reply:
I thought it would be a nice feature to have, especially if you want to scroll down only some amount of lines..

Reply:
Drop these `C-up`/`C-down` mappings, I'd prefer users add these using their local config.